singlestore/README.md

```diff
@@ -0,0 +1,84 @@
+# Agent Check: SingleStore
+
+## Overview
+
+This check monitors [SingleStore][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The SingleStore check is included in the [Datadog Agent][2] package.
```

Comment:
This link doesn't look right, maybe https://app.datadoghq.com/account/settings#agent? Seems like the template doesn't create a link for it

Reply:
Good point, added it